shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared;
+
+import com.google.common.base.Preconditions;
+
+public class MetricsTags {
+
+    // Metric Tag Names
+    public static final String TAG_CONTAINER = "container";
+    public static final String TAG_HOST = "host";
+    public static final String TAG_SCOPE = "scope";
+    public static final String TAG_STREAM = "stream";
+    public static final String TAG_SEGMENT = "segment";
+    public static final String TAG_TRANSACTION = "transaction";
+    public static final String TAG_EPOCH = "epoch";
+
+    private static final String TRANSACTION_DELIMITER = "#transaction.";
+    private static final String EPOCH_DELIMITER = ".#epoch.";
+    /**
+     * The Transaction unique identifier is made of two parts, each having a length of 16 bytes (64 bits in Hex).
+     */
+    private static final int TRANSACTION_PART_LENGTH = Long.BYTES * 8 / 4;
+    /**
+     * The length of the Transaction unique identifier, in bytes (it is made of two parts).
+     */
+    private static final int TRANSACTION_ID_LENGTH = 2 * TRANSACTION_PART_LENGTH;
+
+    /**
+     * Generate string array as container tag for metric.
```

Comment:
The description for these methods could be a bit more descriptive, for example `Generates a tag based on the input containerId to be associated with a metric.`

Reply:
Fixed